### PR TITLE
Correction in Syntax and addition of domain

### PR DESCRIPTION
**Pull request by joinimran**
As mentioned by the user, the domain was not added for cloud-based protection and there was a syntax issue with the smart screen domain.

Problem: https://github.com/MicrosoftDocs/windows-itpro-docs/issues/8876

### CODE REVIEW

**Comment by joinimran** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: windows/privacy/windows-endpoints-20H2-non-enterprise-editions.md [hunk 1]
```diff
@@ -180,6 +181,7 @@ The following methodology was used to derive the network endpoints:
 |||TLSv1.2/HTTPS/HTTP|config.edge.skype.com|
 |Teams|The following endpoint is used for Microsoft Teams application.|TLSv1.2/HTTPS/HTTP|config.teams.microsoft.com|
 |Windows Defender|The following endpoint is used for Windows Defender when Cloud-based Protection is enabled|TLSv1.2/HTTPS|wdcp.microsoft.com|
+||||wdcpalt.microsoft.com|
 |||HTTPS/HTTP|*smartscreen-prod.microsoft.com|
```
Comment (illfated) [inline windows/privacy/windows-endpoints-20H2-non-enterprise-editions.md:185]: Missing dot ( . ) here too:
```suggestion
|||HTTPS/HTTP|*.smartscreen-prod.microsoft.com|
```

File: windows/privacy/windows-endpoints-20H2-non-enterprise-editions.md [hunk 2]
```diff
@@ -245,6 +247,7 @@ The following methodology was used to derive the network endpoints:
...
```
Comment (illfated) [inline windows/privacy/windows-endpoints-20H2-non-enterprise-editions.md:251]: ... and here:
```suggestion
|||HTTPS/HTTP|*.smartscreen-prod.microsoft.com|
```

**Comment by illfated** [comment]
To be honest, I wish the article author could verify if any of the FQDNs should start with only an asterisk ( \* ) without a dot ( . ) before the domain name. There seems to be many more, but I do not know if there are valid cases where the hostname part of the FQDN can be extended with partial names added to the front of the existing FQDN, without a separating dot ( . ) after the asterisk.

**Review by JohanFreelancer9 — APPROVED** [review]
Copy is fine.

**Comment by illfated** [comment]
> To be honest, I wish the article author could verify if any of the FQDNs should start with only an asterisk ( * ) without a dot ( . ) before the domain name. There seems to be many more, but I do not know if there are valid cases where the hostname part of the FQDN can be extended with partial names added to the front of the existing FQDN, without a separating dot ( . ) after the asterisk.

OK, I think I see the intention of an asterisk without a dot preceding the FQDN, it allows for other semi-separating characters like dash( - ), underscore( _ ), and slash/.

File: windows/privacy/windows-endpoints-20H2-non-enterprise-editions.md [hunk 3]
```diff
@@ -261,4 +264,4 @@ The following methodology was used to derive the network endpoints:
 |Xbox Live|The following endpoints are used for Xbox Live.|
 |||TLSv1.2/HTTPS/HTTP|dlassets-ssl.xboxlive.com|
 |||TLSv1.2/HTTPS|da.xboxservices.com|
-|
\ No newline at end of file
+|
```
Comment (illfated) [inline windows/privacy/windows-endpoints-20H2-non-enterprise-editions.md:267]: This line is an orphaned cell divider and is not used in the table (ignored), so it is safe to remove it.
```suggestion

```

**Review by illfated — APPROVED** [review]
The format `*smartscreen.microsoft.com` was reported as incorrect, so I can only approve the changes already made. There are still some remaining typos.

@Gental-Giant : Would you like to confirm?

File: windows/privacy/windows-endpoints-20H2-non-enterprise-editions.md [hunk 4]
```diff
@@ -180,7 +181,8 @@ The following methodology was used to derive the network endpoints:
 |||TLSv1.2/HTTPS/HTTP|config.edge.skype.com|
 |Teams|The following endpoint is used for Microsoft Teams application.|TLSv1.2/HTTPS/HTTP|config.teams.microsoft.com|
 |Windows Defender|The following endpoint is used for Windows Defender when Cloud-based Protection is enabled|TLSv1.2/HTTPS|wdcp.microsoft.com|
-|||HTTPS/HTTP|*smartscreen-prod.microsoft.com|
+||||wdcpalt.microsoft.com|
+|||HTTPS/HTTP|*.smartscreen-prod.microsoft.com|
 ||The following endpoints are used for Windows Defender SmartScreen reporting and notifications.|TLSv1.2|*smartscreen.microsoft.com|
```
Comment (illfated) [inline windows/privacy/windows-endpoints-20H2-non-enterprise-editions.md:186]:
```suggestion
||The following endpoints are used for Windows Defender SmartScreen reporting and notifications.|TLSv1.2|*.smartscreen.microsoft.com|
```

File: windows/privacy/windows-endpoints-20H2-non-enterprise-editions.md [hunk 5]
```diff
@@ -245,7 +247,8 @@ The following methodology was used to derive the network endpoints:
...
```
Comment (illfated) [inline windows/privacy/windows-endpoints-20H2-non-enterprise-editions.md:252]:
```suggestion
||The following endpoints are used for Windows Defender SmartScreen reporting and notifications.|TLSv1.2|*.smartscreen.microsoft.com|
```

**Review by illfated — APPROVED** [review]
Much better, thank you. The solution is coherent with the reported issue.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by illfated** [comment]
#sign-off

**Comment by Gental-Giant** [comment]
#sign-off

**Pull request merged**